b2sdk/file_version.py

```diff
@@ -351,6 +351,7 @@ def _get_upload_headers(self) -> bytes:
             file_retention=self.file_retention,
             legal_hold=self.legal_hold,
             cache_control=self.cache_control,
+            expect_100_continue=False,
```

Comment:
wasn't it supposed to be ON by default with option of disabling?

Reply:
Isn't `file_version` a separate request than an upload request?

Reply:
sorry my bad:)

also this isn't used in any request but  in `has_large_header` from what I see